specs/gltfExtensions/MaxarTemporalLightTraits/MaxarTemporalLightTraitsValidationSpec.ts

```diff
@@ -0,0 +1,144 @@
+import { validateGltf } from "../validateGltf";
+
+describe("MAXAR_temporal_light_traits extension validation", function () {
+  it("detects no issues in validTemporalLightTraits", async function () {
+    const result = await validateGltf(
+      "./specs/data/gltfExtensions/khrLightsPunctual/maxarTemporalLightTraits/validTemporalLightTraits.gltf"
+    );
+    // Should only have info messages, no errors
+    let hasErrors = false;
+    for (let i = 0; i < result.length; i++) {
+      if (result.get(i).severity === "ERROR") {
+        hasErrors = true;
+        break;
+      }
+    }
+    expect(hasErrors).toBe(false);
+  });
+
+  it("detects missing waveform property in temporal light traits", async function () {
+    const result = await validateGltf(
+      "./specs/data/gltfExtensions/khrLightsPunctual/maxarTemporalLightTraits/invalidMissingWaveform.gltf"
+    );
+    expect(result.length).toBeGreaterThan(0);
+
+    // Find the error about missing waveform property
```

Comment:
Such inconvenient checks should not be necessary. See final/summarizing review comment